test/qcqpmodel.jl

```diff
@@ -355,3 +355,23 @@ context("With solver $(typeof(solver))") do
     @fact solve(modQ) --> :Optimal
     @fact getObjectiveValue(modQ) --> roughly(-0.75, 1e-6)
 end; end; end
+
+facts("[qcqpmodel] Rotated second-order cones") do
+for solver in soc_solvers
```

Comment:
``soc_solver`` won't work since ECOS and SCS don't support rotated second-order cones